src/cljs/org/broadinstitute/firecloud_ui/main.cljs

```diff
@@ -140,6 +140,39 @@
       (when (:show-nih-link-warning? props)
         [nih-link-warning/NihLinkWarning])])})
 
+;; GAWB-666 Globally show Queued and Cromwell (active) counts
+(defn queue-status-counts [workflow-counts]
+  (let [count-map (get workflow-counts "workflowCountsByStatus" (empty {}))]
+    {:queued (+ (get count-map "Queued" 0) (get count-map "Launching" 0))
+     :active (+ (get count-map "Submitted" 0) (get count-map "Running" 0) (get count-map "Aborting" 0))
+     :terminal (+ (get count-map "Failed" 0) (get count-map "Succeeded" 0) (get count-map "Aborted" 0))}))
```

Comment:
Terminal isn't currently in use in the UI, but is a category of statuses defined in Rawls. See https://github.com/broadinstitute/rawls/blob/e6be332405bfa36c35f935d7ea613e46a11feca8/src/main/scala/org/broadinstitute/dsde/rawls/model/ExecutionModel.scala#L294